cypress-custom/integration/fee.test.ts

```diff
@@ -66,53 +77,26 @@ describe('Fetch and persist fee', () => {
     _assertFeeFetched(DAI)
   })
 
-  // TODO: not sure if it's easy to test this
   it('Re-fetched when it expires', () => {
-    const NETWORK = ChainId.RINKEBY.toString()
-    const TOKEN = 'ETH'
-    const VALUE = JSON.stringify({
-      [NETWORK]: {
-        [TOKEN]: {
-          fee: {
-            minimalFee: '777',
-            feeRatio: 777,
-            expirationDate: new Date(Date.now() - 200000).toISOString()
-          },
-          token: TOKEN
-        }
-      }
-    })
-    // GIVEN: A fee is present in the local storage
-    // Set expiring fee in localStorage
-    _getLocalStorage()
-      .then($storage => {
-        // set time in the past
-        $storage.setItem(FEE_QUOTES_LOCAL_STORAGE_KEY, VALUE)
-      })
-      .its(FEE_QUOTES_LOCAL_STORAGE_KEY)
-      // Keep retrying until localStorage is populated from app
-      .should($feeStorage => expect($feeStorage).to.have.property(NETWORK))
-      // Check that fee is currently in the PAST
-      .then($feeStorage => {
-        // we need to parse JSON
-        const feeStorage = JSON.parse($feeStorage)
-        const NOW = new Date()
-        const feeExpirationDate = new Date(feeStorage[NETWORK].ETH.fee.expirationDate)
-
-        // Here we expect the current saved expirationDate to be in the PAST
-        expect(NOW).to.be.greaterThan(feeExpirationDate)
-      })
-      // WHEN: When the fee quote expires, we refetch the fee
-      // better imo than using cy.wait - clear fee storage
-      .then(() => _getLocalStorage().then($storage => $storage.removeItem(FEE_QUOTES_LOCAL_STORAGE_KEY)))
-      .its(FEE_QUOTES_LOCAL_STORAGE_KEY)
-      // THEN: We get another quote
-      .should($feeStorage => {
-        const feeStorage = JSON.parse($feeStorage)
-
-        const NOW = new Date()
-        const feeExpirationDate = new Date(feeStorage[NETWORK].ETH.fee.expirationDate)
-        expect(feeExpirationDate).to.be.greaterThan(NOW)
+    // const ETH = 'ETH'
+    const FEE = {
+      // expiration date now + 8 hours
+      expirationDate: new Date(Date.now() + FOUR_HOURS * 2),
+      minimalFee: '0',
+      feeRatio: 0
+    }
+
+    cy.stubResponse({ query: FEE_QUERY, alias: 'feeRequest', body: FEE })
+
+    // WHEN: The user comes back 4h later (so the fee quote is expired)
+    cy.tick(FOUR_HOURS)
+
+    cy.wait('@feeRequest')
```

Comment:
Where is the THEN? 

sth like?
`// THEN: The fee is automatically re-fetched`